### ISSUE TITLE
Can't pull from Jitpack

### ISSUE THREAD

**Issue by Brumstar** [phishman3579/android-quick-response-code#5]
I'm trying to pull your library into an Android studio project through the build.gradle configurations mentioned. However, your project never successfully gets pulled and the dependencies fail to get resolved. When I go to https://jitpack.io/com/github/phishman3579/android-quick-response-code/tag/build.log, I can see that none of your builds have ever succeeded and there's nothing for me to pull. I saw that you closed another issue because you didn't have time to support this project, but I'd really like to use it and I was wondering if I could at least fix the builds.

**Comment by phishman3579** [comment]
Yes, please do fix the build. That's the reason I open sourced this project, if I don't have time to fix a problem then maybe some one else will. If you fix the build, please submit a pull request and I'll accept.

**Comment by Brumstar** [comment]
I have a fix for the build but I need to be able to pull request it. Do you need to add me as a collaborator for me to do that?

**Comment by phishman3579** [comment]
If you have forked from this repository using git, you shouldn't need to be a collaborator to submit a pull request.
https://yangsu.github.io/pull-request-tutorial/

**Comment by Brumstar** [comment]
Thank you for the tutorial link.
